module/account/src/Persistence/Dbal/Projector/User/UserCreatedEventProjector.php

```diff
@@ -60,7 +60,7 @@ public function __invoke(UserCreatedEvent $event): void
                 'language' => $event->getLanguage()->getCode(),
                 'password' => $event->getPassword()->getValue(),
                 'is_active' => $event->isActive(),
-                'avatar_filename' => $event->getAvatarFilename() ? $event->getAvatarFilename()->getValue() : null,
+                'avatar_filename' => $event->getAvatarFilename(),
```

Comment:
Is always user created with avatar ???

Reply:
no but method getAvatarFilename can return null or string